pink/rondis/Makefile

```diff
@@ -28,11 +33,18 @@ ifndef RONDB_PATH
   exit 1
 endif
 RONDB_INCLUDE_DIR=$(RONDB_PATH)/include/storage/ndb
+ifeq ($(UNAME_S),Darwin)
+RONDB_LIBRARY=$(RONDB_PATH)/lib/libndbclient.dylib
+else
 RONDB_LIBRARY=$(RONDB_PATH)/lib/libndbclient.so
+endif
 
 CXXFLAGS+= -I$(PINK_INCLUDE_DIR) -I$(SLASH_INCLUDE_DIR) -I$(RONDB_INCLUDE_DIR)
 
 DEP_LIBS = $(PINK_LIBRARY) $(SLASH_LIBRARY) $(RONDB_LIBRARY)
+ifeq ($(UNAME_S),Darwin)
+DEP_LIBS+= /opt/homebrew/lib/libprotobuf.dylib
```

Comment:
Wouldn't Brew dependencies be included automatically?

Reply:
For some reason not, so something to figure out why that doesn't happen